mayavi/core/base.py

```diff
@@ -438,32 +450,33 @@ def __view_filename_default(self):
         class_filename = module[-1] + '.py'
         module_dir_name = module[1:-1]
         base_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-        view_filename = os.path.join(*([base_dir] + module_dir_name \
-                                        + UI_DIR_NAME + [class_filename]))
+        view_filename = os.path.join(*([base_dir] + module_dir_name
+                                       + UI_DIR_NAME + [class_filename]))
         return view_filename
 
-
     def __module_view_default(self):
         """ Try to load a view for this object.
         """
         view_filename = self._view_filename
-        try:
-            result = importlib.load_module('view', open(view_filename, 'r'),
-                            view_filename, ('.py', 'U', 1))
-            view = result.view
-        except:
+        if os.path.exists(view_filename):
```

Comment:
It looks like this is the only functional change here - is that right? (I see a lot of other changes, but at a glance they all look cosmetic; I want to be sure I'm not missing something else significant.)

Reply:
Yes indeed, this is the only non trivial change, the rest were PEP8 errors.  Thank you!